scripts/dts/extract/reg.py

```diff
@@ -28,12 +28,36 @@ def extract(self, node_address, names, def_label, div):
 
         node = reduced[node_address]
         node_compat = get_compat(node_address)
+        binding = get_binding(node_address)
 
         reg = reduced[node_address]['props']['reg']
         if type(reg) is not list: reg = [ reg, ]
 
         (nr_address_cells, nr_size_cells) = get_addr_size_cells(node_address)
 
+        if 'parent' in binding:
```

Comment:
The flow mean: Son will use parent description and create MACRO? Eg follow code: the enc28j60 own cs-gpios ?
```
&spi1 {
	status = "ok";
	cs-gpios = <&gpio0 7 0>;

	enc28j60@0 {

	};
};
```

Reply:
@lgl88911 current the CS information should be specified at the controller node level in dts, not at the SPI device level.

Reply:
Base the pr , flow DS is right? If it right, the dts python script will handle the change and no need modify the dts binding yaml (like #12641)
&spi1 {
	status = "ok";
	enc28j60@0 {
             cs-gpios = <&gpio0 7 0>;
	};
};

Reply:
@lgl88911 See #12226.  #12641 is the wrong solution to the problem.